gui-libs/libdecor/libdecor-0.2.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A client-side decorations library for Wayland clients"
+HOMEPAGE="https://gitlab.freedesktop.org/libdecor/libdecor"
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/libdecor/libdecor.git"
+	inherit git-r3
+else
+	SRC_URI="https://gitlab.freedesktop.org/libdecor/libdecor/-/archive/${PV}/${P}.tar.bz2"
+	KEYWORDS="~amd64"
+fi
+LICENSE="MIT"
+SLOT="0"
+IUSE="+dbus +gtk examples"
+
+RDEPEND="
+	>=dev-libs/wayland-1.18
+	x11-libs/pango
+	x11-libs/cairo
+	dbus? ( sys-apps/dbus )
+"
+
+DEPEND="
+	${RDEPEND}
+	>=dev-libs/wayland-protocols-1.15
+"
+
+BDEPEND="
+	gtk? ( x11-libs/gtk+:3 )
+	examples? (
+		media-libs/libglvnd
+		x11-libs/libxkbcommon
+	)
+"
```

Comment:
I missed that you've moved these into `BDEPEND` since your first attempt. You seem to misunderstand what `BDEPEND` is for. It is for tools that are needed by the system doing the building, which may be a different architecture to the system that will eventually run the package. All of these should be in `DEPEND` and `RDEPEND`. Meson would be in `BDEPEND`, but that's covered by the eclass. You should also add `virtual/pkgconfig` and `dev-util/wayland-scanner` though.

Reply:
Oh ok, since that depends are use to build the exemples that why I add them there

Reply:
Done